docs/_getting-started-macos-ios.md

```diff
@@ -22,9 +22,7 @@ If you have already installed Node on your system, make sure it is Node 18 or ne
 
 ### Xcode
 
-The easiest way to install Xcode is via the [Mac App Store](https://itunes.apple.com/us/app/xcode/id497799835?mt=12). Installing Xcode will also install the iOS Simulator and all the necessary tools to build your iOS app.
-
-If you have already installed Xcode on your system, make sure it is version 14.1 or newer.
+The easiest way to install Xcode is via the [Mac App Store](https://itunes.apple.com/us/app/xcode/id497799835?mt=12). Installing Xcode will also install the iOS Simulator and all the necessary tools to build your iOS app. Please use the latest version of Xcode.
```

Comment:
```suggestion
The easiest way to install Xcode is via the [Mac App Store](https://itunes.apple.com/us/app/xcode/id497799835?mt=12). Installing Xcode will also install the iOS Simulator and all the necessary tools to build your iOS app. Please use the **latest version** of Xcode.
```

or let's put it in a new paragraph as it's likely to be skipped being so late in the paragraph

Reply:
+1 to this.